File: one.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int t = in.nextInt();
        int[] ar = new int[t];
        for(int a0 = 0; a0 < t; a0++){
            int n = in.nextInt();
            n--;
            int threes = 3*(n/3)*(n/3 + 1)/2;
            int fives = 5*(n/5)*(n/5 + 1)/2;
            int fifteens = 15*(n/15)*(n/15 + 1)/2;
            int count = threes + fives - fifteens;
            System.out.println(count);
        }
        in.close();
    }
}

File: one.py
#!/bin/python3

import sys

# Take in num of nums
t = int(input().strip())
for a0 in range(t):
    # access first num, but subtract one since we only want natural number below it
    n = int(input().strip()) - 1
    # Grabbing summation of 3's and 5's possible in one step
    # This is because all summations follow pattern of (0, 3, 6, 9...)
    # can use summations of numbers below n then multiply by 3/5/15
    # Because 3 and 5 both count multiples of 15, we must subtract it once from our final
    threes = int(3*(n//3)*(n//3 + 1)//2)
    fives = int(5*(n//5)*(n//5 + 1)//2)
    fifteens = int(15*(n//15)*(n//15 + 1)//2)
    result = threes + fives - fifteens;
    print(result)

